### ISSUE TITLE
Improve the text to give a message

### ISSUE THREAD

**Issue by GaelVaroquaux** [covid19-dash/covid-dashboard#27]
The goal of this visualization is to get people to understand the local growth dynamics and to push them to change their behavior. It should be clear about the limitation of the numbers and of the extrapolation. The text that we have is still a draft.

**Comment by poldrack** [comment]
writing this accurately will require knowing more details about the forecasting models...

**Comment by GaelVaroquaux** [comment]
> writing this accurately will require knowing more details about the forecasting models...

I should probably do a large part of it.

However, we need to list more general epidemic facts, such as things like (list to be populated as we think more):

* it takes many days (a week?) for the dynamics to change given the incubation time

* the extrapolation does not take in account any public health or medical measure that will be implemented in the future

**Comment by poldrack** [comment]
yeah, that's the point at which I think we really need an epi expert on board...

**Comment by GaelVaroquaux** [comment]
Leave this open, just in case someone with the relevant expertise sees it.
